jetbrains-rider/it/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotnetLocalLambdaRunConfigurationIntegrationTest.kt

```diff
@@ -78,7 +75,6 @@ abstract class DotnetLocalLambdaRunConfigurationIntegrationTestBase(private val
     }
 
     @Test
```

Comment:
what did these @TestEnvironments do? Or what were they supposed to do?

Reply:
It was used to set the solution to use, but we inject it as a parameter from the child class